get-started/index.md

```diff
@@ -248,38 +246,38 @@ auth implementation.
 
 ### [Deploy an Application](#deploy-an-application)
 For demo purposes, let's go ahead and deploy a sample application to our
-OpenShift cluster in the same namespace as our Cryostat instance. If you have
-deployed Cryostat into a namespace where you are already running other
+**OpenShift** cluster in the same namespace as our **Cryostat** instance. If you have
+deployed **Cryostat** into a namespace where you are already running other
```

Comment:
"namespace" in these two lines should be either **bolded** or `monospaced`, and should probably be Capitalized as well